docs/index.html

```diff
@@ -3,9 +3,13 @@
 <head>
   <meta charset="utf-8" />
   <title>PokéSprite</title>
+  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>

```

Comment:
For consistency, add a space before the closing tag, like the other `<link>` items.